src/styles/components/TextEditor.styl

```diff
@@ -0,0 +1,121 @@
+.TextEditor
```

Comment:
There is a bunch of extra space below everything on the project detail page with this branch. I'm not sure what's causing it.